controllers/authController.js

```diff
@@ -0,0 +1,72 @@
+const User = require('../models/User')
+const bcrypt = require('bcryptjs');
+const {validationResult} = require('express-validator')
+const jwt = require('jsonwebtoken')
+const {secret} = require('../config/config')
+
+const generateAccessToken = (id) => {
+	const payload = {id}
+	return jwt.sign(payload, secret, {expiresIn: '24h'})
+}
+
+class authController {
+	async registration(req, res) {
+		try {
+			//const validationErrors = validationResult(req)
+			if (!validationResult(req).isEmpty()) return res.status(400).json({message: `Registration errors ${validationErrors}`})
```

Comment:
With the `if` statement always use curly braces
```suggestion
			if (!validationResult(req).isEmpty()) {
			  return res.status(400).json({message: `Registration errors ${validationErrors}`})
			}
```